ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -21,6 +21,12 @@ parameters:
 - name: BASE_DOMAIN
   value: origin-ci-int-aws.dev.rhcloud.com
   required: true
+- name: IMAGE_EGRESS_HTTP_PROXY
+  required: true
+  value: http://registry.svc.ci.openshift.org/origin/4.2:egress-http-proxy
+- name: PROXY_ENABLED
+  required: true
+  value: true
```

Comment:
This is fine for testing, but you'll want to turn it off before this PR lands.  To get a proxy job, you'll want a new job like [this][1] that explicitly sets `PROXY_ENABLED` (or whatever the knob ends up being).

[1]: https://github.com/openshift/release/pull/3877/files#diff-faaaaebefcc08c8105ee06d2eaefba12R353